src/Type/ItemPrice.php

```diff
@@ -240,18 +247,28 @@ private function amountTaxAll()
      * @param array $tax_group A subset of the taxes array
      * @param float $taxable_price The total amount from which to calculate tax
      * @param TaxPrice $tax A specific tax from the group whose tax amount to retrieve (optional)
+     * @param string $type The type of tax for which to retrieve amounts (inclusive, exclusive, or inclusive_calculated)
      * @return float The total tax amount for all taxes set for this item in this group, or
      *  the tax amount for the given TaxPrice
      */
-    private function compoundTaxAmount(array $tax_group, $taxable_price, TaxPrice $tax = null)
+    private function compoundTaxAmount(array $tax_group, $taxable_price, TaxPrice $tax = null, $type = null)
```

Comment:
TaxPrice already has the type so this is a little odd.

Reply:
This is meant for the times when TaxPrice is not submitted and thus we are evaluating all taxes on the price.